package.json

```diff
@@ -39,19 +39,21 @@
     "linter"
   ],
   "dependencies": {
-    "commander": "2.9.0",
-    "gherkin": "4.0.0",
-    "glob": "7.0.5",
-    "lodash": "4.13.1"
+    "commander": "^2.17.1",
```

Comment:
Can you remove the use of `^`? I prefer to pin dependencies and not use `^` to lower the chances of having this package break because a dependency released a bad version.